api_v3/lib/types/report/KalturaReportFilter.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * @package api
+ * @subpackage objects
+ */
+class KalturaReportFilter extends KalturaObject
+{
+	/**
+	 * @var string
```

Comment:
Please add description to define the content and structure of this string, it's needed for the auto-generated documentation.